main_test.go

```diff
@@ -257,7 +315,7 @@ func TestNewMessage(t *testing.T) {
 		Title:      mockConfig.CardTitle,
 		Summary:    fmt.Sprintf("%v #%v succeeded", mockConfig.AppTitle, mockConfig.BuildNumber),
 		// Be mindful of list order
-		Sections: []Section{primarySection, buildSuccessFacts},
+		Sections: []Section{primarySection, buildSuccessFacts, imagesSection},
```

Comment:
"Be mindful of list order" 😉 

Since the `imagesSection` was moved to be before the facts section in `main.go`, the test expects the same order.

Reply:
Ah man, I could've sworn I ran the full test suite, but it must've just been the images section test. Fixing this now.